v3-nginx/Dockerfile

```diff
@@ -84,34 +89,33 @@ COPY --from=build /usr/share/TLS/server.key /etc/nginx/conf/server.key
 COPY --from=build /usr/share/TLS/server.crt /etc/nginx/conf/server.crt
 COPY ./tls.conf /etc/nginx/conf.d/tls.conf.disabled
 
-RUN export version=$(echo `/usr/sbin/nginx -v 2>&1` | cut -d '/' -f 2) && \
-    wget --quiet http://nginx.org/download/nginx-$version.tar.gz && \
-    tar -xvzf nginx-$version.tar.gz && \
-    cd /nginx-$version && \
+RUN version="$(/usr/sbin/nginx -v 2>&1 | cut -d '/' -f 2)" && \
+    wget --quiet http://nginx.org/download/nginx-"$version".tar.gz && \
+    tar -xvzf nginx-"$version".tar.gz && \
+    cd /nginx-"$version" && \
     ./configure --with-compat --add-dynamic-module=../ModSecurity-nginx && \
     make modules && \
     cp objs/ngx_http_modsecurity_module.so /etc/nginx/modules
 
 RUN mkdir /etc/modsecurity.d && \
-    cd /etc/modsecurity.d && \
-    echo "include \"/etc/modsecurity.d/modsecurity.conf\"" > setup.conf && \
-    wget --quiet https://raw.githubusercontent.com/SpiderLabs/ModSecurity/v3/master/modsecurity.conf-recommended && \
-    mv modsecurity.conf-recommended modsecurity.conf && \
-    wget --quiet https://raw.githubusercontent.com/SpiderLabs/ModSecurity/v3/master/unicode.mapping
+    echo "include \"/etc/modsecurity.d/modsecurity.conf\"" > /etc/modsecurity.d/setup.conf && \
+    wget --quiet https://raw.githubusercontent.com/SpiderLabs/ModSecurity/v3/master/modsecurity.conf-recommended \
+      -o /etc/modsecurity.d/modsecurity.conf && \
+    wget --quiet https://raw.githubusercontent.com/SpiderLabs/ModSecurity/v3/master/unicode.mapping \
+      -o /etc/modsecurity.d/unicode.mapping
 
 RUN sed -i '1iload_module modules/ngx_http_modsecurity_module.so;' /etc/nginx/nginx.conf && \
     sed -i -e 's/http {/http {\n    modsecurity on;\n    modsecurity_rules_file \/etc\/modsecurity.d\/setup.conf;\n/g' /etc/nginx/nginx.conf
 
 ENV LD_LIBRARY_PATH /lib:/usr/lib:/usr/local/lib
 
-RUN apt-get install -y vim
-RUN echo $SERVERNAME
+RUN echo "$SERVERNAME"
 RUN sed -i -E "s/server_name .*;/server_name $SERVERNAME;/g" /etc/nginx/conf.d/default.conf
 
 RUN if [ "$SETTLS" = "True" ]; then \
       echo "setting TLS"; \
       mv /etc/nginx/conf.d/tls.conf.disabled /etc/nginx/conf.d/tls.conf; \
-      sed -i -E 's/server_name .*;/server_name $SERVERNAME;/g' /etc/nginx/conf.d/tls.conf; \
+      sed -i -E "s/server_name .*;/server_name $SERVERNAME;/g" /etc/nginx/conf.d/tls.conf; \
```

Comment:
Using double-quotes will potentially allow to expand the globbing (`.*`) in the string. Is that intended?

Reply:
Well, without quoting it won't work as intended:
```bash
[fzipi@railgun-lan example]$ cat nginx.conf 
server_name example.com;
[fzipi@railgun-lan example]$ SERVERNAME=globbing.com
[fzipi@railgun-lan example]$ sed -E 's/server_name .*;/server_name $SERVERNAME;/g' nginx.conf 
server_name $SERVERNAME;
[fzipi@railgun-lan example]$ sed -E "s/server_name .*;/server_name $SERVERNAME;/g" nginx.conf
server_name globbing.com;
```
You can do also any of:
```bash
[fzipi@railgun-lan example]$ sed -E "s/server_name .*;/server_name "$SERVERNAME";/g" nginx.conf 
server_name globbing.com;
[fzipi@railgun-lan example]$ sed -E "s/server_name .*;/server_name "${SERVERNAME}";/g" nginx.conf 
server_name globbing.com;
```

Reply:
True. The double-quotes are needed for expanding the `$SERVERNAME`.